scripts/optimize_images/static-images.ts

```diff
@@ -91,11 +94,31 @@ const caseStudyLogos = chain(allCaseStudies)
   }))
   .value();
 
-const cmsImages = concat(
+const cmsStructuredImages = concat(
   profilePictures,
   governmentLogos,
   partnersLogos,
   caseStudyLogos,
 );
 
-export default concat(staticImages, cmsImages);
+const cmsStructuredImageFilenames = cmsStructuredImages.map(
+  imageInfo => imageInfo.inputPath,
+);
+
+const allCmsImages = fs
+  .readdirSync(absolutePath('public/images_cms'))
+  .filter(isRasterImage)
+  .map(imgFilename => path.join('public/images_cms', imgFilename));
+
+const remainingCmsImages = difference(
+  allCmsImages,
+  cmsStructuredImageFilenames,
+);
+
+const cmsRemainingImages = remainingCmsImages.map((imagePath: string) => ({
```

Comment:
Can you add a comment (perhaps borrowing from the PR description) about these images and the chosen 600px size?